package.json

```diff
@@ -31,6 +31,9 @@
     "jest": "^26.6.3",
     "lerna": "2.4.0",
     "micromatch": "^4.0.2",
+    "metro": "*",
+    "metro-config": "*",
+    "metro-babel-register": "*",
```

Comment:
This is necessary because otherwise eslint will complain. This change has no other effect since those modules are already linked as part of Yarn's workspaces.